README.md

```diff
@@ -206,17 +182,13 @@ Firstly you need to open the browser at the link <http://localhost:8091> and edi
 Zwave settings:
 
 - **Serial port**: The serial port where your controller is connected
-- **Network key** (Optional): Zwave network key if security is enabled. The correct format is `"0xCA,0xFE,0xBA,0xBE,.... "` (16 bytes total)
-- **Logging**: Enable/Disable Openzwave Library logging
-- **Save configuration**: Store zwave configuration in `zwcfg_<homeHex>.xml` and `zwscene.xml` files this is needed for persistent node information like node name and location
-- **Poll interval**: Interval in milliseconds between polls (should not be less than 1s per device)
+- **Network key** (Optional): Zwave network key if security is enabled. The correct format is `"CAFEBABE.... "` (32 chars total)
```

Comment:
Some guidance on translating would be handy and/or accept both and transform it

Reply:
About network key, @AlCalzone could you provide me an example valid value?

Reply:
16 bytes HEX. You can take an OZW network key and remove all occurences of `0x` or `, `.